src/Pickles/Pickles.Test/WhenParsingCommandLineArguments.cs

```diff
@@ -232,6 +232,112 @@ public void ThenCanParseResultsFileAsSemicolonSeparatedListThatEndsWithASemicolo
         }
 
         [Test]
+        public void ThenCanParseMultipleResultsFilesWithWildCard()
+        {
+            FileSystem.AddFile(@"c:\results1.xml", "<xml />");
+            FileSystem.AddFile(@"c:\results2.xml", "<xml />");
+            var args = new[] { @"-link-results-file=c:\results*.xml" };
+
+            var configuration = new Configuration();
+            var commandLineArgumentParser = new CommandLineArgumentParser(FileSystem);
+            bool shouldContinue = commandLineArgumentParser.Parse(args, configuration, TextWriter.Null);
+
+            Check.That(shouldContinue).IsTrue();
+            Check.That(configuration.HasTestResults).IsTrue();
+            Check.That(configuration.TestResultsFiles.Count()).IsEqualTo(2);
```

Comment:
Thanks for the unit tests. I like your test cases.

Here's a way to write the assertions with more focus on their intention:

`Check.That(configuration.TestResultsFiles.Select(trf => trf.FullName)).ContainsExactly(@"c:\results1.xml", @"c:\results2.xml");`

Please change that here and in the other test cases as well.